tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -35,7 +35,7 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 USE_CHAIN_DMA=true,NUM_SDW_AMP_LINKS=2,SDW_SPK_STREAM=SDW1-Playback,SDW_SPK_IN_STREAM=SDW1-Capture,\
 SDW_DMIC_STREAM=SDW0-Capture"
 
-"cavs-sdw\;sof-mtl-rt713-l0-rt1316-l12-rt1713-l3\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=2,SDW_DMIC=1"
+"cavs-sdw\;sof-mtl-rt713-l0-rt1316-l12-rt1713-l3\;PLATFORM=mtl,USE_CHAIN_DMA=true,NUM_SDW_AMP_LINKS=2,SDW_DMIC=1"
```

Comment:
@bardliao We can do this in steps (multiple PRs), but we need to enable USE_CHAIN_DMA in all topoplogies unless there's a very strong reason not do. This enables bitstream passthrough for display audio.

Reply:
> @bardliao We can do this in steps (multiple PRs), but we need to enable USE_CHAIN_DMA in all topoplogies unless there's a very strong reason not do. This enables bitstream passthrough for display audio.

Can we set USE_CHAIN_DMA = true by default? Like
```patch
diff --git a/tools/topology/topology2/cavs-rt5682.conf b/tools/topology/topology2/cavs-rt5682.conf
index f3cdc8d47aeb..227a839c19ac 100644
--- a/tools/topology/topology2/cavs-rt5682.conf
+++ b/tools/topology/topology2/cavs-rt5682.conf
@@ -69,7 +69,7 @@ Define {
        HDMI4_ID                        6
        SPK_ID                          7
        PLATFORM                        "none"
-       USE_CHAIN_DMA                   "false"
+       USE_CHAIN_DMA                   "true"
        DEEP_BUFFER_PIPELINE_ID         15
        DEEP_BUFFER_PCM_ID              31
        DEEP_BUFFER_PIPELINE_SRC        'mixin.15.1'
diff --git a/tools/topology/topology2/cavs-sdw.conf b/tools/topology/topology2/cavs-sdw.conf
index a33e60e92d6d..64d88a3eae13 100644
--- a/tools/topology/topology2/cavs-sdw.conf
+++ b/tools/topology/topology2/cavs-sdw.conf
@@ -63,7 +63,7 @@ Define {
        DMIC0_ID 4
        DMIC1_ID 5
        DMIC0_PCM_CAPS 'Gain Capture 13'
-       USE_CHAIN_DMA   "false"
+       USE_CHAIN_DMA   "true"
        DEEP_BUFFER_PIPELINE_ID         15
        DEEP_BUFFER_PCM_ID              31
        DEEP_BUFFER_PIPELINE_SRC        'mixin.15.1'
diff --git a/tools/topology/topology2/sof-hda-generic.conf b/tools/topology/topology2/sof-hda-generic.conf
index 1bd06bff796f..c732463c4aff 100644
--- a/tools/topology/topology2/sof-hda-generic.conf
+++ b/tools/topology/topology2/sof-hda-generic.conf
@@ -43,7 +43,7 @@ Define {
        HDA_CONFIG  "none"
        PLATFORM "none"
        NUM_DMICS 0
-       USE_CHAIN_DMA   "false"
+       USE_CHAIN_DMA   "true"
 # doesn't it need to define DMIC0_HOST_PIPELINE_SINK and DMIC0_DAI_PIPELINE_SRC?
        DMIC0_HOST_PIPELINE_SINK 'gain.11.1'
        DMIC0_DAI_PIPELINE_SRC 'module-copier.12.2'
```

Reply:
We should use HDMI_USE_CHAIN_DMA to make sure we don't make any confusions. HDaudio can also be handled with chain_dma.

Also this should be a global property for MTL, not something that is changed in each variant.

Reply:
All theses cases need to be updated with NUM_HDMI=0 removed and CHAIN_DMA set:


https://github.com/thesofproject/sof/pull/8527/files/417e167c0688a83ae2f95f1aa2fdec553ec02e59#diff-0ac40f848737bb3a9eceac9f3a1f9d86436f053a08378a28777cc779d2467f54R23

https://github.com/thesofproject/sof/pull/8527/files/417e167c0688a83ae2f95f1aa2fdec553ec02e59#diff-0ac40f848737bb3a9eceac9f3a1f9d86436f053a08378a28777cc779d2467f54R27

https://github.com/thesofproject/sof/pull/8527/files/417e167c0688a83ae2f95f1aa2fdec553ec02e59#diff-0ac40f848737bb3a9eceac9f3a1f9d86436f053a08378a28777cc779d2467f54R40

https://github.com/thesofproject/sof/pull/8527/files/417e167c0688a83ae2f95f1aa2fdec553ec02e59#diff-0ac40f848737bb3a9eceac9f3a1f9d86436f053a08378a28777cc779d2467f54R43

https://github.com/thesofproject/sof/pull/8527/files/417e167c0688a83ae2f95f1aa2fdec553ec02e59#diff-0ac40f848737bb3a9eceac9f3a1f9d86436f053a08378a28777cc779d2467f54R47

No reason to special-case a subset of SoundWire topologies